migrations/2_deploy_contracts.js

```diff
@@ -28,7 +28,7 @@ module.exports = (deployer, network, accounts) => {
     );
   }
 
-  if (network == "kovan") {
+  if (network === 'kovan') {
     deployer.deploy([
       Exchange,
       // Redeem,
```

Comment:
yup - most change looks good esp [this](https://github.com/melonproject/protocol/blob/master/migrations/2_deploy_contracts.js#L53-L58)